lstchain/calib/camera/sampling_interval_coefficient_calculate.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from astropy.io import fits
+from numba import jit, prange
+
+from ctapipe.core import Component
+
+
+__all__ = ['SamplingIntervalCalculate']
+
+N_PIXELS = 1855
```

Comment:
you can import these from `ctapipe_io_lst.constants.`

Reply:
done!